packages/fake-driver/package.json

```diff
@@ -43,6 +43,7 @@
     "prepare": "npm run build",
     "test": "npm run test:unit",
     "test:e2e": "mocha --timeout 20s --slow 10s \"./test/e2e/**/*.spec.js\"",
+    "test:smoke": "node ./build/lib/index.js",
```

Comment:
any reason for all of these that we're specifying a specific js file rather than say `node .` and relying on resolution through the `main` field (which is theoretically what users are also doing?)

Reply:
yeah, we could use `node .` in _most_ places, but some of them (like `fake-driver`) actually start a server or do some extra work we don't need.  I think I mentioned this...somewhere. maybe it was in my head

Reply:
> We have to be careful to avoid doing unnecessary work, so doctor and fake-driver do not load the entry point (as those are CLI executables). appium does load its entry point (main.js), but only to run --show-build and exit.